packages/core/test-helpers/core-test-helpers-test-utils/index.ts

```diff
@@ -6,9 +6,4 @@
  * Side Public License, v 1.
  */
 
-export { createHttpServer } from '@kbn/core-http-server-mocks';
-export { setupServer } from './integration_tests/saved_objects/routes/test_utils';
-export {
-  getDeprecationsFor,
-  getDeprecationsForGlobalSettings,
-} from '@kbn/core-test-helpers-deprecations-getters';
+export { setupServer } from './src/setup_server';
```

Comment:
should we also export `createExportableType`?

Reply:
No. The utility's only used in the saved objects integration tests. This PR handles the root-level `test_utils` file.